README.md

```diff
@@ -44,13 +43,11 @@ In this To-do-List project, a simple HTML page that displays list of To Do tasks
 
```

Comment:
![image](https://github.com/tan12082001/Project-To-Do-List/assets/93039359/5f71bc8e-82a4-43a9-829e-a00b3211200a)

**[OPTIONAL REQUIREMENTS]**

- [ ] It would be great if you remove the title **(Project-To-Do-List)** and description because you already have a title and description of the project below the Table of Contents so kindly remove the title and description which is above the Table of Contents to make your readme more descriptive and professional and look clean 👍
- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍